### ISSUE TITLE
"view on Github" banner is a bit misleading

### ISSUE THREAD

**Issue by TehShrike** [sdmp/sdmp.github.io#44]
The "view on Github" diagonal text in a serif font evokes the "edit this post" link of [Noddity](http://noddity.com/), but is functionally different (it doesn't open the current post to be edited, it just takes you to the repo's readme)

Github's banner is well-understood to take you to the repo's home page, though it's not as trendy to use as it used to be.  Maybe consider [Github Corners](http://tholman.com/github-corners/) as an alternative?

**Comment by saibotsivad** [comment]
Yeah, that's a good point. The "view on Github" was actually copied from Noddity's "edit this post" style, and it doesn't mean anything particular since it doesn't link to the page it's on.

I'm torn between removing it entirely, and changing it so it links to the correct markdown file...

**Comment by TehShrike** [comment]
I actually like the idea of a global "take me to the Github project" icon for projects like this.  I would just prefer it evoke the Github banners more than Noddity's post-editing link.
